### PR TITLE
Use Pedersen DKG in CreateLTS

### PR DESCRIPTION
**Pull request by kc1212**
This is the first step for #1536 where we implement the Pedersen DKG in an onet protocol and use it in CreateLTS. We need to use this DKG because key-resharing is not available for the Rabin DKG.

### CODE REVIEW

File: dkg/pedersen/dkg.go [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+	}, nil
+}
+
+// Start sends the Announce-message to all children
+func (o *Setup) Start() error {
+	log.Lvl3("Starting Protocol")
+	// 1a - root asks children to send their public key
+	errs := o.Broadcast(&Init{Wait: o.Wait})
+	if len(errs) != 0 {
+		return fmt.Errorf("boradcast failed with error(s): %v", errs)
```
Comment (Gilthoniel) [inline dkg/pedersen/dkg.go]: broadcast

Comment (jeffallen) [reply dkg/pedersen/dkg.go]:
```suggestion
		return fmt.Errorf("broadcast failed with error(s): %v", errs)
```

Comment (jeffallen) [reply dkg/pedersen/dkg.go]: Perhaps like that you can just click and accept in Github? It's a new feature, unclear how it works...

Comment (kc1212) [reply dkg/pedersen/dkg.go]: wow that's a useful feature

File: dkg/DKG.md [hunk 2]
```diff
@@ -5,20 +5,32 @@ Distributed Key Generation
...
-Based Cryptosystems" by R. Gennaro, S. Jarecki, H. Krawczyk, and T. Rabin.
-DKG enables a group of participants to generate a distributed key
-with each participants holding only a share of the key. The key is also
-never computed locally but generated distributively whereas the public part
-of the key is known by every participants.
+Under this folder, we implement two distributed key generation protocols:
+Pedersen DKG and Rabin DKG.
+
+DKG enables a group of participants to generate a distributed key with each
+participants holding only a share of the key. The key is also never computed
```
Comment (jeffallen) [inline dkg/DKG.md]:
```suggestion
participant holding only a share of the key. The key is also never computed
```

**Pull request merged**